ssl/ssl_lib.c

```diff
@@ -3084,6 +3084,14 @@ void ssl_set_masks(SSL *s)
 
     if (rsa_enc || rsa_sign) {
```

Comment:
Interesting the (preexisting) mismatch in this function as to whether braces are used for one-line conditional bodies.